ext/http/00_serve.ts

```diff
@@ -384,8 +388,10 @@ class CallbackContext {
   /** @type {Promise<void> | undefined} */
   closing;
   listener;
+  asyncContext;
 
   constructor(signal, args, listener) {
+    this.asyncContext = getAsyncContext();
```

Comment:
Please remind me - does it mean that there's some overhead added no matter if OTEL is enabled or not?

Reply:
technically yes but when we benchmarked it, it was not measurable.